### PR TITLE
Stackhpc/alt-1 openstack pike magnum queens

### PR DESCRIPTION
**Pull request by brtkwr**
```
source venv/kayobe/bin/activate
source ../kayobe-config/kayobe-env
cd src/kayobe
export KAYOBE_VAULT_PASSWORD=****secret****
```

Only for provisioning a local VM for the first time:
	
```
kayobe overcloud host configure
```

Add public key for  `$USER` into `authorized_keys` for `stack` user on all hosts:
```
10.41.253.[100: seed | 101: prod controller | 102: prod monitor | 103: alt-1 controller]
```

To build and deploy new magnum image:
```
kayobe overcloud container image build magnum --push -e kolla_source_version=stable/queens -e kolla_openstack_release=6.0.0.0 -e kolla_source_url=https://git.openstack.org/openstack/kolla
kayobe overcloud container image pull --kolla-tags magnum
kayobe overcloud service upgrade --kolla-tags magnum 
```

Only on the local VM:

```
kayobe overcloud service reconfigure --kolla-tags haproxy

### CODE REVIEW

File: etc/kayobe/kolla.yml [hunk 1]
```diff
@@ -12,11 +12,15 @@
 
 # URL of Kolla source code repository if type is 'source'.
 #kolla_source_url:
+kolla_source_url: https://git.openstack.org/openstack/kolla
 
 # Version (branch, tag, etc.) of Kolla source code repository if type is
 # 'source'.
 #kolla_source_version:
 
+# Uncomment for building magnum/6.0.0.0 image
```
Comment (dougszumski) [inline etc/kayobe/kolla.yml]: I think you can override it on the command line with ```-e kolla_source_version=stable/queens```. It's probably best to do it that way. Then you can have a one liner for building the queens magnum image, and not have to bother uncommenting stuff in here. It also helps keep the config clean.

Comment (markgoddard) [reply etc/kayobe/kolla.yml]: Nice point. Kayobe power user!

Comment (dougszumski) [reply etc/kayobe/kolla.yml]: I'm looking forward to my ewok sticker.

File: etc/kayobe/kolla.yml [hunk 2]
```diff
@@ -12,6 +12,7 @@
 
 # URL of Kolla source code repository if type is 'source'.
 #kolla_source_url:
+kolla_source_url: https://git.openstack.org/openstack/kolla
```
Comment (markgoddard) [inline etc/kayobe/kolla.yml]: The default in kayobe is https://github.com/stackhpc/kolla, let's keep that.

File: etc/kayobe/kolla/passwords.yml [hunk 3]
```diff
@@ -1,1010 +1,1010 @@
 $ANSIBLE_VAULT;1.1;AES256
```
Comment (markgoddard) [inline etc/kayobe/kolla/passwords.yml]: Did you find out what these changes are?

**Review by markgoddard — CHANGES_REQUESTED** [review]
Needs merge conflicts resolving.

File: etc/kayobe/kolla/passwords.yml (hunk 3 above)
Comment (brtkwr) [reply etc/kayobe/kolla/passwords.yml]: A password was updated automatically during the build by the looks of it

Comment (markgoddard) [reply etc/kayobe/kolla/passwords.yml]: Which one?

File: etc/kayobe/kolla.yml (hunk 2 above)
Comment (brtkwr) [reply etc/kayobe/kolla.yml]: Do we need this source specifically for building magnum for P3?

File: etc/kayobe/kolla/passwords.yml (hunk 3 above)
Comment (brtkwr) [reply etc/kayobe/kolla/passwords.yml]: https://github.com/stackhpc/alaska-kayobe-config/blob/7edd6fddf80789a8ce677573e1734a0469979efb/etc/kayobe/kolla/passwords.yml

Comment (markgoddard) [reply etc/kayobe/kolla/passwords.yml]: Which password though?

File: etc/kayobe/kolla.yml (hunk 2 above)
Comment (markgoddard) [reply etc/kayobe/kolla.yml]: Yes - we don't have the stable/queens in our fork

File: etc/kayobe/kolla/passwords.yml (hunk 3 above)
Comment (dougszumski) [reply etc/kayobe/kolla/passwords.yml]: If it helps, I normally I do something like ```ansible-vault view passwords.yml > new.yml```, git stash, or checkout the original file (making a backup first), then ```ansible-vault view passwords.yml > old.yml``` and then ```diff new.yml old.yml``` to see which passwords have been changed.

**Review by markgoddard — APPROVED** [review]
Ok, I think we're good.

**Pull request merged**